apps/gateway/lib/gateway/auth/google_token_manager.ex

```diff
@@ -0,0 +1,23 @@
+defmodule Gateway.Auth.GoogleTokenManager do
+  @moduledoc """
+  Module responsible to verify and validate the Google TokenIDs.
+  Its logic has been done following the GoogleCertificates docs:
+  https://hexdocs.pm/google_certs/readme.html
+  """
+
+  use Joken.Config, default_signer: nil
+
+  @iss "https://accounts.google.com"
+
+  defp aud, do: System.get_env("GOOGLE_CLIENT_ID")
```

Comment:
Let's get this from configuration rather than env vars

Reply:
Done! Let me know if it should be different.